src/components/login/login.css

```diff
@@ -3,26 +3,27 @@
   flex-direction: column;
   justify-content: center;
   width: 350px;
-  height: 650px;
-  border: 1px black solid;
-  border-radius: 45px;
-  background-color: rgb(238, 246, 253);
+  height: 75%;
+  border: 0.1rem black solid;
+  border-radius: 4.5rem;
+  background-color: #eef6fd;
 }
 .login__popup {
   display: flex;
   flex-direction: column;
   width: 100%;
   align-items: center;
-  color: red;
-  gap: 15px;
+  color: #ff0000;
+  gap: 20px;
 }
 .login__button {
   width: 80%;
+  height: 3.8rem;
```

Comment:
Moim zdaniem nie warto definiować wysokości na przycisku. Powinno to dziać się samo za pomocą definiowania paddingu wewnątrz buttonu + wysokosci tekstu. Co gdy tekst będzie łamał się do drugiej linijki? W przypadku ustawionego statycznie heighta bedzie to wygladalo zle, jesli bedzie to padding + font-size + line-height to mozesz tam dać X linijek i bedzie git.